### ISSUE TITLE
Add example dataset to this repository

### ISSUE THREAD

**Issue by lisphilar** [lisphilar/covid19-sir#17]
**Is your feature request related to a problem? Please describe.**
To try this package, it is necessary to prepare a dataset in advance. This sometimes prevent new users to use this package.

**Describe the solution you'd like**
Include Japanese dataset to this package.
[Kaggle: COVID-19 dataset in Japan](https://www.kaggle.com/lisphilar/covid19-dataset-in-japan) is maintained by me and this can be included in this repository.

**Comment by lisphilar** [comment]
Dataset in Japan was included in issue26_lisphilar branch and this is not included in master branch.
Pull request of Japanese dataset to master branch is included in #30 pull request.

**Comment by lisphilar** [comment]
Japan dataset is included data directory of this repository.

`covsirphy.cleaning.data_loader.DataLoader` was created for automatic data downloading in version 2.2.4.

Example codes are as follows.
```Python
import covsirphy as cs
# Set the directory to save the datasets
data_loader = cs.DataLoader("input")
# The number of cases in Japan
japan_data = data_loader.japan()
print(japan_data.citation)
jpn_df = japan_data.cleaned()
print(jpn_df)
```
